{
  try {
    return runMainWithExitCode(args,projectRoot);
  }
 catch (  HumanReadableException e) {
    Console console=new Console(stdOut,stdErr,new Ansi());
    console.printFailure(e.getHumanReadableErrorMessage());
    return 1;
  }
}
